.editorconfig

```diff
@@ -4,15 +4,11 @@ root = true
 charset = utf-8
 indent_size = 2
 indent_style = space
-quote_type = double
+tabSize=2
+quote_type = single
 
 insert_final_newline = true
-max_line_length = 80
+max_line_length=120
 trim_trailing_whitespace = true
-
 [*.md]
-trim_trailing_whitespace = false
-
-[*.ts]
-indent_style = tab
-max_line_length = 120
+trim_trailing_whitespace = falses
```

Comment:
Reminding, here is a typo error: `falses` -> `false`

Reply:
thanks!